### ISSUE TITLE
Add GUI to help show what test step execution graph

### ISSUE THREAD

**Issue by jentfoo** [threadly/ambush#1]
This library has the potential to have very complicated execution graphs (if desired!). It would be great to draw these graphs so that people can easily visualize the execution process. This will help in debugging possible issues as well.

**Comment by jentfoo** [comment]
This has started to be implemented.  I need to document it but you need to go into the ambushGUI directory and executed ./gradlew build....

java -cp build/libs/* org.threadly.load.ScriptGraphBuilder script.factory.to.call key1=value1 key2=value2....

**Comment by jentfoo** [comment]
I just improved this graph GUI intensely.  Features I just added:
  * Step names are hidden if there is a large qty of steps
    * If names are hidden, they can be easily seen by hovering your mouse over a point
  * The visibility of step names can be adjusted via a button at the top left corner
  * If there is a lot of steps either wide (in parallel) or long (sequential), the graph is zoomed in
    * A preview window draws a box where your currently viewing the graph
    * The area of view can be adjusted multiple ways:
      * You can drag in the main window an area where there is no point (draging a point moves that point)
      * You can double click in the preview window to center at that point
      * You can drag the box in the preview window

It has many other features to make it nice that are less obvious.  If any GUI guru's out there have comments for improvement, please provide them.  I am trying to make it very useful and clear to understand.

I am closing this specific issue as done, though I did open issue #9 to add documentation before our first stable release.  I opened another issue because lots needs documentation besides this.
